Assets/HoloToolkit-Examples/WindowsMRControllers-GrabMechanics/Textures/MRTK_Logo.png.meta

```diff
@@ -1,18 +1,20 @@
 fileFormatVersion: 2
-guid: e7c1f473df573ff4094e55c183149d96
```

Comment:
Do we want this in the grab mechanics examples folder, or actually have it in it's own folder in Examples with other toolkit branding assets?